packages/interactor/package.json

```
@@ -16,7 +16,7 @@
   "scripts": {
     "lint": "eslint '{src,test}/**/*.ts'",
     "mocha": "mocha -r ts-node/register",
-    "test:unit": "mocha -r ts-node/register test/**/*.test.ts",
+    "test:unit": "mocha -r ts-node/register 'test/**/*.test.ts'",
```

@jbolda So something like this?
```suggestion
    "test:unit": "mocha -r ts-node/register \"test/**/*.test.ts\"",
```

---

Yep! I'm sure there are so a handful of single quotes hanging around after my PR. I was focusing on just getting tests running (and linting was likely ignored).